src/main/provisio/maven-distro.xml

```diff
@@ -12,7 +12,7 @@
     <artifactSet to="ext" ref="runtime.classpath" />
   </artifactSet>
 
-  <fileSet to="/conf/logging">
+  <fileSet to="/">
```

Comment:
Can you elaborate on this? Is this change of behaviour in provisio or you want to move the files inside the distro?

Reply:
Setting "to" attribute value has no effecti on proviso 0.1.16  and results in wrong directory structure on provision 0.1.56